protobuf-kafkaconnect-converter/README.md

```diff
@@ -28,4 +28,28 @@ Following optional properties can be used to change Protobuf schema inference fr
 | Connect metadata | Usage |
 |:--------------------------|:------------------|
 |protobuf.tag | Specify the Protobuf tag number to use for a field. Fields are generated sequentially otherwise.
-|protobuf.type | Specify the Protobuf data type to use for converting a certain field. Default mappings are used otherwise. |
\ No newline at end of file
+|protobuf.type | Specify the Protobuf data type to use for converting a certain field. Default mappings are used otherwise. |
+|protobuf.package | Specify the Protobuf package name defined in the Protobuf message. |
+
+
+### Protobuf to Connect conversion
+
+| Protobuf Type         | Connect schema type         |
+|:----------------------|:----------------------------|
+|INT32                                                    |INT32 (default), INT8, INT16|
+|SINT32, SFIXED32                                         |INT32                       |
+|INT64, SINT64, UINT64, SFIXED64, FIXED64, FIXED32, UINT32|INT64                       |
+|FLOAT                                                    |FLOAT32                     |
+|DOUBLE                                                   |FLOAT64                     |
+|BOOL                                                     |BOOLEAN                     |
+|STRING                                                   |string                      |
+|BYTES                                                    |bytes                       |
+|ENUM                                                     |STRING                      |
+|repeated                                                 |ARRAY                       |
+|map                                                      |map                         |
+|message                                                  |struct                      |
+|oneof                                                    |struct                      |
+|google.type.TimeOfDay (message)                          |TIME                        |
+|google.protobuf.Timestamp (message)                      |TIMESTAMP                   |
+|Decimal.proto ([Custom implementation](https://github.com/protocolbuffers/protobuf/issues/4406)) (message)          |DECIMAL                     |
```

Comment:
Decimal and few others will have a follow up CR, correct ?

Reply:
Yes.